### ISSUE TITLE
Category tile changes the whole time and causes memory problems

### ISSUE THREAD

**Issue by nickyhaze** [intershop/intershop-pwa#245]
When we open the category page we noticed that our browser had some issues because of an infinite loop.

We did some research and saw that the children() inside createCategoryView is called an enormous amount of times.

To reproduce this:

1. go to CategoryTileComponent
2. checkout the changes by implementing ngOnChanges inside this component. 
```
ngOnChanges(c: SimpleChanges) {
    console.log('c', c);
  }
```

We have also included a screenshot to make this more clear. It would be great if you can have a look at this. Thank you in advance!

![image](https://user-images.githubusercontent.com/13309873/82321859-d17dfc80-99d5-11ea-9160-004ecbc467e7.png)

**Comment by rkarl-ish** [comment]
Hi Nicky.  We will have a look at this after the 0.20 release.

**Comment by dhhyi** [comment]
@nickyhaze we have now integrated the refactoring and it will be available in the upcoming version [0.22](https://github.com/intershop/intershop-pwa/releases/tag/0.22.0). There should be significantly less view updates now 😉

**Comment by nickyhaze** [comment]
Great, thank you!
